### DISCUSSION TITLE
GitHub Student Pack Rejection

### DISCUSSION THREAD

**Discussion by Gamius00** [community/community#59979] [GitHub Education]
### Hi everyone,

Hello Guys, 

I sent a many requests to GitHub for my GitHub Developer Student Pack. 
But always the requests was rejected. Here is the Error 
![image](https://github.com/community/community/assets/127024349/e8410dcb-0caf-4b86-9f5b-c3d5d48f56a0)

Here is my Testimony: 
![Zeugnis]()

MODERATOR EDIT: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by samus-aran** [comment]
Hi @Gamius00 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
